databases/jdp/database.go

```diff
@@ -222,14 +242,38 @@ func creditFromIdAndMetadata(id string, md Metadata) credit.CreditMetadata {
 	return crd
 }
 
-func trimFileSuffix(filename string) string {
-	for _, suffix := range supportedSuffixes {
-		trimmedFilename, trimmed := strings.CutSuffix(filename, "."+suffix)
-		if trimmed {
-			return trimmedFilename
+// creates a Frictionless DataResource-savvy name for a file:
+// * the filename is down-cased
```

Comment:
=== lowercased?

Reply:
Yes. For what it's worth. I think in my own mental pidgen I was distinguishing between the verb "to downcase" and the adjective "lowercased".

Reply:
just wanted to check we were talking about the same thing!